modules/ROOT/pages/install-self-managed.adoc

```diff
@@ -0,0 +1,144 @@
+= Install Runtime Fabric on Self-managed Kubernetes
+
+You can install Anypoint Runtime Fabric into a Kubernetes cluster provided by AWS Elastic Kubernetes Service (EKS) or Azure Kubernetes Service (AKS). 
+
+== Before you Begin
+
+Before installing Anypoint Runtime Fabric in a self-managed Kubernetes service, ensure the following:
+
+* You have installed and configured your Kubernetes cluster as follows:
++
+- Running Kubernetes version 1.14.x to 1.18.x.
+- Running an ingress controller to send external requests to applications.
+
+== Create a Runtime Fabric using Runtime Manager
+
+To install Runtime Fabric on a self-managed Kubernetes cluster, first create a Runtime Fabric using Runtime Manager. This is required to obtain the activation data which is needed during installation.
+
+. From Anypoint Platform, select Runtime Manager.
+. Click *Runtime Fabrics*.
+. Click *Create Runtime Fabric*.
+. Enter the name of the new Runtime Fabric, then select one of the following options:
++
+* Amazon Elastic Kubernetes Service
+* Azure Kubernetes Service
+
+. Click *Next*.
+. Review the *Support responsibility* disclaimer, then if you agree click *Continue*.
++
+Runtime Manager creates the Runtime Fabric and displays the Activation State page. This page displays the activation data used to install Runtime Fabric on a Kubernetes service. Copy this data to the clipboard for use in the next section. 
+
+
+== Install Runtime Fabric on a Self-Managed Kubernetes Cluster
+
+After creating a Runtime Fabric and getting the activation data, install Runtime Fabric into your Kubernetes service using the `rtfctl` command.
+
+[NOTE]
+====
+If your Kubernetes configuration is not located in the `~/.kube/config` directory, set the `KUBECONFIG` environment variable before running `rtfctl`:
+
+----
+export KUBECONFIG=<path-to-kubeconfig>
+----
+====
+
+
+. Download the `rtfctl` command:
++
+`rtfctl` is supported on Windows, MacOS (Darwin), and Linux. Download this utility using the URLs below:
++
+*Windows:*
++
+----
+curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl-windows/latest -o rtfctl
```

Comment:
the output file name for windows platform will need `.exe`:
`curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl-windows/latest -o rtfctl.exe`

Reply:
Thanks @chingyi-lin! I added the extension.